content/automate/ManageIQ/Infrastructure/VM/Transform/Import.class/__methods__/list_storages.rb

```diff
@@ -23,6 +23,7 @@ def main
                       values_hash[nil] = 'None'
                     else
                       provider.storages.each do |storage|
+                        next unless storage.storage_domain_type == "data"
                         values_hash[storage.id] = storage.name
```

Comment:
Instead of `next unless` this could be a simple if-statement.

`values_hash[storage.id] = storage.name if storage.storage_domain_type == "data"`